OpenTokSamples/7.Multiparty-ConstraintLayout/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+
+    defaultConfig {
+        applicationId "com.tokbox.multiparty_constraintlayout"
+        minSdkVersion 19
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:25.3.1'
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
+    compile 'pub.devrel:easypermissions:0.3.1'
+    compile 'com.opentok.android:opentok-android-sdk:2.10.+'
+}
```

Comment:
I see you are bumping up versions here and we are leaving the ones we have in the other samples. IMHO I'd would be great to file a ticket to take care of bumping up the version in the other samples.